design/comments.go

```diff
@@ -152,6 +152,10 @@ var _ = a.Resource("work-item-comments", func() {
 			a.GET("comments"),
 		)
 		a.Description("List comments associated with the given work item")
+		a.Params(func() {
+			a.Param("page[offset]", d.String, "Paging start position")
```

Comment:
Pagination is the same everywhere. Tracking https://github.com/almighty/almighty-core/issues/697